packages/flet_flashlight/lib/src/create_control.dart

```diff
@@ -0,0 +1,20 @@
+import "package:flet/flet.dart";
+
+import 'flash.dart';
+
+CreateControlFactory createControl = (CreateControlArgs args) {
+  switch (args.control.type) {
+    case "flashlight":
+      return FlashControl(
```

Comment:
Also can you rename `FlashControl` to `FlashlightControl`?

Reply:
also fix this or answer to me

Reply:
Fixed.